src/app/routes/utils/parseAvRoute/index.ts

```diff
@@ -61,8 +61,8 @@ const LANGS = [
   'tr',
   'uk',
   'ur',
-  'uz',
```

Comment:
I'd keep this in to continue support for older media content when Uzbek wasn't a dual-publishing service.